source/scss/variables/_typography-variables.scss

```diff
@@ -60,6 +60,15 @@ $header-bookmark-line-height: .75rem;
 $header-hotline-font-size: $smaller-font-size;
 $header-hotline-line-height: $font-size-base;
 
+// Linked Price
+// Used for Offer Teaser and Multi Offer Teaser
+$linked-price-font-size: $font-size-base * 1.375;
+$linked-price-line-height: $font-size-base * 1.625;
+$linked-price-font-weight: $font-weight-bold;
+$linked-price-letter-spacing: .3px;
+$linked-price-info-font-size: $font-size-base * .75;
+$linked-price-info-line-height: $font-size-base * 1;
```

Comment:
Was this '1' added here for unit conversion on Sass?

Reply:
It symbolises the 16px. We want to bind the font-size to the font-size-base, which is 16px or 1rem. Whenever you change the size of this font-size you just simply need to change the number. The output is 1rem in our css file. So this is really for maintainability.